pkg/services/container/morph/executor.go

```diff
@@ -274,3 +291,41 @@ func (s *morphExecutor) GetExtendedACL(_ context.Context, body *container.GetExt
 
 	return res, nil
 }
+
+func (s *morphExecutor) validateToken(t *sessionV2.Token, cID *refs.ContainerID, op sessionV2.ContainerSessionVerb) error {
+	c := t.GetBody().GetContext()
+	cc, ok := c.(*sessionV2.ContainerSessionContext)
+	if !ok {
+		return errors.New("session is not container related")
+	}
+
+	if verb := cc.Verb(); verb != op {
+		return fmt.Errorf("wrong container session operation: %s", verb)
```

Comment:
error should not contain input imo cuz client knows what he sent (same for other cases)

Reply:
> client knows what he sent

it sounds to me "error doer knows that he does an error". it can be a bug inside our node, or it can be a bug on the client side, i do not think it is critical to mention that session has a verb that does not relate to the operation it is attached to